libnestutil/regula_falsi.h

```diff
@@ -39,7 +39,7 @@ namespace nest
  */
 template < typename CN >
 double
-regula_falsi( CN* node, const double dt )
+regula_falsi( CN& node, const double dt )
```

Comment:
Can't this be a `const` reference? I don't think we change the `node` in any way?

Reply:
Sorry, I was a bit quick and forgot the `const`.